controllers/auth.js

```diff
@@ -27,9 +28,15 @@ const authenticate = (req, res) => {
       message: 'Not Found',
       data: { error: 'Incorrect password' }
     });
-  } else {
+  } else if (hardcodedUser.name !== username) {
+    res.status(404).send({
+      code: 404,
+      message: 'Not Found',
+      data: { error: 'Incorrect username' }
+    });
+  } else if (username === hardcodedUser.name && password === hardcodedUser.password) {
```

Comment:
There is no necessity in checking for username === & password === since it is done above. By duplicating logic you increase complexity and decrease readability

Reply:
@AliyaMansurova please take a look

Reply:
fixed